app/models/track.rb

```diff
@@ -8,4 +8,14 @@ class Track < ActiveRecord::Base
   validates_presence_of :download_url
   validates_presence_of :creator_id
   validates_uniqueness_of :track_name, case_sensitive: false, scope: :creator_id
+
+  before_save :sanitize_url
+
+  def sanitize_url
+    return if self.download_url.nil?
+    unless self.download_url.include?("http://") || self.download_url.include?("https://")
```

Comment:
Line is too long. [90/80]<br>Use a guard clause instead of wrapping the code inside a conditional expression.<br>Redundant `self` detected.